content/installation/linux/LinuxRepoConfig.md

```diff
@@ -7,7 +7,7 @@ tags: "installation linux package repo distribution"
 
 ## About Linux Package Distribution
 
-Some of Contrast's products, including the [Webserver agent](installation-webserver.html#webserver-install), are distributed through Linux packages. This makes installation and updates very easy and familiar as it utilizes the Linux packaging and distribution system. 
+Some of Contrast's products, including the [Proxy agent](installation-proxy.html#proxy-install), are distributed through Linux packages. This makes installation and updates very easy and familiar as it utilizes the Linux packaging and distribution system. 
```

Comment:
please verify that this fixes the broken link. I'm not sure what the actual value should be so I guessed.